src/HexagonOffload.cpp

```diff
@@ -284,7 +292,7 @@ class InjectHexagonRpc : public IRMutator {
 }
 
 Stmt inject_hexagon_rpc(Stmt s, const Target &host_target) {
-    Target target = hexagon_remote_target;
+    Target &target = hexagon_remote_target;
```

Comment:
This seems wrong, it's going to modify the hexagon_remote_target, which is suppose to be a globally reused constant. 

Really, that should just not be a global, we should just change this code to be:

    Target target(Target::NoOS, Target::Hexagon, 32);
    for (Target::Feature i ...)